http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/HttpWsmDao.scala

```diff
@@ -185,11 +186,20 @@ class HttpWsmDao[F[_]](httpClient: Client[F], config: HttpWsmDaoConfig)(implicit
                                                          false
       )
       aksSubnetName <- getLandingZoneResourceName(groupedLzResources, "DeployedSubnet", AKS_NODE_POOL_SUBNET, false)
-      postgresName <- getLandingZoneResourceName(groupedLzResources,
+      postgresResource <- getLandingZoneResource(groupedLzResources,
                                                  "Microsoft.DBforPostgreSQL/flexibleServers",
-                                                 SHARED_RESOURCE,
-                                                 false
+                                                 SHARED_RESOURCE
       ).attempt
+      postgresServer = postgresResource.toOption.flatMap { resource =>
+        getLandingZoneResourceName(resource, false).map { pgName =>
+          val tagValue = getLandingZoneResourceTagValue(resource, "pgbouncer-enabled")
+          val pgBouncerEnabled: Boolean = java.lang.Boolean.parseBoolean(tagValue.getOrElse("false"))
+          logger.info(
+            s"Landing Zone Postgres server has 'pgbouncer-enabled' tag $tagValue; setting pgBouncerEnabled to $pgBouncerEnabled."
+          )
+          PostgresServer(pgName, pgBouncerEnabled)
+        }
+      }
```

Comment:
the log message here will be a little spammy, since this is also called when deleting apps and when creating runtimes - but I'd rather be spammy than not have the tools we need to debug.